src/WebJobs.Script.Grpc/Abstractions/IWorkerProcessFactory.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Diagnostics;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Abstractions.Rpc
+{
+    public interface IWorkerProcessFactory
```

Comment:
What's the conceptual difference between an RpcServer (i.e. IRpcServer) and a "Worker"? Are they the same? Should these types be IRpcServerFactory, RpcServerConfig, etc.?

Reply:
IRpcServer is an entity which listens for incoming rpc connections on the host side. IWorkerProcessFactory is something I'm trying to change, but basically can be thought of as the code which is creating the child processes which start up rpc clients that connect to IRpcServer.